data/themes/default/style.css

```diff
@@ -278,33 +283,33 @@ QScrollBar::add-page:vertical:pressed, QScrollBar::sub-page:vertical:pressed {
 /* scrollbar: handles (sliders) */
 
 QScrollBar::handle:horizontal {
-	background: #3f4750;
+	background: rgb(183, 217, 195);
```

Comment:
This seems to add another variant of "LMMS green" into the mix. I propose to use one of the existing green colors so that the different elements have a consistent styling and a discernible style.